runatlantis.io/docs/command-requirements.md

```diff
@@ -152,18 +153,19 @@ Applies to `merge` checkout strategy only.
 
 #### Usage
 You can set the `undiverged` requirement by:
-1. Creating a `repos.yaml` file with the `apply_requirements` key:
+1. Creating a `repos.yaml` file with `apply_requirements` and `import_requirements` keys:
    ```yaml
    repos:
    - id: /.*/
      apply_requirements: [undiverged]
+     import_requirements: [undiverged]
    ```
 1. Or by allowing an `atlantis.yaml` file to specify the `apply_requirements` key in your `repos.yaml` config:
    #### repos.yaml
     ```yaml
     repos:
     - id: /.*/
-      allowed_overrides: [apply_requirements]
+      allowed_overrides: [apply_requirements, apply_requirements]
```

Comment:
Did you mean

```suggestion
      allowed_overrides: [apply_requirements, import_requirements]
```

Reply:
thanks!, all docs feedback is commited into https://github.com/runatlantis/atlantis/pull/2783/commits/aabc2609246a1338b42dee8662c6f70d26d78cea